grammars/hrm.cson

```diff
@@ -32,29 +32,25 @@
         'name': 'keyword.control.c'
   }
   {
-    'match': '^    (COMMENT) +([0-9]+)'
+    'match': '^ *(COMMENT) +([0-9]+)'
     'captures':
       '1':
         'name': 'entity.name.function.preprocessor.c'
       '2':
-        'name': 'string.quoted.double.include.c'
+        'name': 'constant.numeric.c'
```

Comment:
Comments are referenced via 'constant numeric' id's, I felt it was more appropriate. I was wondering if you chose that particular syntax type for color or there was more meaning behind it.